test/e2e/suite/serving/cainjector.go

```diff
@@ -412,46 +430,38 @@ var _ = framework.CertManagerDescribe("CA Injector", func() {
 		},
 		getCAs: func(obj runtime.Object) [][]byte {
 			crd := obj.(*apiext.CustomResourceDefinition)
-			if crd.Spec.Conversion == nil || crd.Spec.Conversion.WebhookClientConfig == nil {
+			if crd.Spec.Conversion == nil || crd.Spec.Conversion.Webhook == nil || crd.Spec.Conversion.Webhook.ClientConfig == nil {
 				return nil
 			}
-			return [][]byte{crd.Spec.Conversion.WebhookClientConfig.CABundle}
+			return [][]byte{crd.Spec.Conversion.Webhook.ClientConfig.CABundle}
 		},
 		disabled: "ConversionWebhook feature not yet enabled on test infra",
 	})
 
-	// TODO: re-enable this test.
-	// This test has been disabled in order to reduce flakes on e2e tests
-	// (hitting around 70% failure rate). This is because when running this test
-	// will cause tiller install to fail, therefore making other tests to fail
-	// when running bother at the same time. We should find a way to make this test run in serial.
-	// https://github.com/jetstack/cert-manager/issues/2353
-	// https://github.com/jetstack/cert-manager/issues/2354
```

Comment:
These remarks talk about these tests causing problems with tiller.
Well AFAIK, we now use a version of Helm which does not use tiller, and I'm not sure if we use Helm in the E2E tests anyway.
I'll re-run the tests a few times to check for any flakiness.

Reply:
Fixes: https://github.com/jetstack/cert-manager/issues/2354